tests/pipelines/ner/test_umls.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+pytest.importorskip("umls_downloader", reason="umls_downloader package not found")
+
+import os  # noqa: E402
+from pathlib import Path  # noqa: E402
+
+from spacy.language import Language  # noqa: E402
+
+from edsnlp.pipelines.ner.umls.patterns import get_path, get_patterns  # noqa: E402
+from edsnlp.utils.examples import parse_example  # noqa: E402
+
+examples = [
+    "Grosse <ent umls=C0010200>toux</ent> : Le malade a été mordu "
+    "<ent umls=C1257901>par</ent> des <ent umls=C0002668>Amphibiens</ent>"
```

Comment:
In the future, it would be interesting to detect these false positives by detecting acronyms in UMLS and matching them separately in uppercase, or perhaps by detecting common adverbs and empty words imo